lblod/app-mandatees-decisions
# Lmb 1053: create an ldes stream instead of consuming all data and producing interesting types

## Description

Remove the producer and all services that where needed to produce the interesting types data. Replace this with an LDES that can be consumed by other apps.

## How to test

1. Run the app
2. When running with docker-compose dev only consumer-0 will run (connected to dev-harvester where our decision is added)
3. This data should be added to the public ldes-stream that can be found in _/data/ldes-feed/public_ 
4. Once the ldes looks ok the feature is working

## Links to other PR's

- https://github.com/lblod/mandataris-service/pull/66
- https://github.com/lblod/app-lokaal-mandatenbeheer/pull/287
- https://mandaten-besluiten.lblod.info/streams/ldes/public/1 (deployed version of this PR)

## Other
